interfaces/builtin/kubernetes_support.go

```diff
@@ -67,17 +67,23 @@ profile systemd_run (attach_disconnected,mediate_deleted) {
   # Common rules for kubernetes use of systemd_run
   #include <abstractions/base>
 
-  /usr/bin/systemd-run r,
+  /{,usr/}bin/systemd-run rm,
   owner @{PROC}/@{pid}/stat r,
   owner @{PROC}/@{pid}/environ r,
   @{PROC}/cmdline r,
+  @{PROC}/sys/kernel/osrelease r,
 
   # setsockopt()
   capability net_admin,
 
+  # ptrace 'trace' is coarse and not required for using the systemd private
+  # socket, and while the child profile omits 'capability sys_ptrace', skip
+  # for now since it isn't strictly required.
+  deny ptrace trace peer=unconfined,
   /run/systemd/private rw,
-  /bin/true ixr,
-  ptrace trace peer=unconfined,
+
+  /{,usr/}bin/true ixr,
```

Comment:
not a blocker, but out of curiosity, why is this in the policy in kubernetes-support to begin with? shouldn't something as common as  `/bin/true` be in the base apparmor template?

Also FWIW, /bin/true exists in both core and core18, so we could add it to the base template I think

Reply:
@anonymouse64 - it is already in the base template. This is in the systemd_run which the child profile that doesn't use that template. The apparmor base abstraction could conceivably use it, as you mention, but we would need to carry it here regardless until everyone upgraded to the new apparmor.